matRad_calcParticleDoseMCsquare.m

```diff
@@ -182,16 +182,35 @@
         dij.doseGrid.x,dij.doseGrid.y',dij.doseGrid.z,'linear');
 end
 
-%BDL File
-bdFile = [machine.meta.machine '.txt'];
-
-% bdFile = 'BDL_matRad.txt'; %use for baseData fit 
-MCsquareBDL = MatRad_MCsquareBaseData(machine,stf);
-%matRad_createMCsquareBaseDataFile(bdFile,machine,1);
-% MCsquareBDL = MCsquareBDL.saveMatradMachine('test');
-MCsquareBDL = MCsquareBDL.writeMCsquareData([MCsquareFolder filesep 'BDL' filesep bdFile]);
-%movefile(bdFile,[MCsquareFolder filesep 'BDL/' bdFile]);
-% MCsquareBDL = MCsquareBDL.saveMatradMachine('testMachine');
+
+% switch for using extising BDL file, or fit one using MCsquareBDL
+useExisting = 1;
```

Comment:
This here I don't really understand. Why is there a hardcoded 1 here?

Ah, I might understand now, that the changes regarding the emittance structure are not in here. We probably should put this on hold until the restructured emittance is merged into this.

Reply:
The problem here is that one needs to prevent the standard BDL file fitting process in a fitting run. This is why the switch is here. With the latest commit I turned it back to zero to ensure normal functionality in all other cases apart from the fitting use. In addition to that I added a comment in the fitting example, explaining the need to change the switch accordingly for the fitting process. Do you have an idea how to solve this problem in a more elegant way?